mkosi/__init__.py

```diff
@@ -5131,27 +5131,54 @@ def parse_ssh_agent(value: str) -> Optional[Path]:
         die(f"SSH agent socket {sock} is not an AF_UNIX socket")
     return sock
 
+USAGE = """
+       mkosi [options...] {b}summary{e}
+       mkosi [options...] {b}build{e} [script parameters...]
+       mkosi [options...] {b}shell{e} [command line...]
+       mkosi [options...] {b}boot{e} [nspawn settings...]
+       mkosi [options...] {b}qemu{e} [qemu parameters...]
+       mkosi [options...] {b}ssh{e} [command line...]
+       mkosi [options...] {b}clean{e}
+       mkosi [options...] {b}serve{e}
+       mkosi [options...] {b}bump{e}
+       mkosi [options...] {b}genkey{e}
+       mkosi [options...] {b}help{e}
+       mkosi -h | --help
+       mkosi --version
+""".format(b=MkosiPrinter.bold, e=MkosiPrinter.reset)
 
 def create_parser() -> ArgumentParserMkosi:
-    parser = ArgumentParserMkosi(prog="mkosi", description="Build Bespoke OS Images", add_help=False)
+    parser = ArgumentParserMkosi(
+        prog="mkosi",
+        description="Build Bespoke OS Images",
+        usage=USAGE,
+        add_help=False)
```

Comment:
```suggestion
        add_help=False,
    )
```
This way it would be more in line with the rest.

Reply:
Thanks, updated.